src/NServiceBus.AcceptanceTesting/Support/ScenarioConfigSource.cs

```diff
@@ -70,15 +70,6 @@ public T GetConfiguration<T>() where T : class, new()
                 }
             }
 
-            if (type == typeof(Logging))
```

Comment:
I don't think this code did anything.
We're using the `ContextAppenderFactory`: `LogManager.UseFactory(new ContextAppenderFactory());` in the Scenario setup.
The ContextAppenderFactory uses the log level configured on the `ScenarioContext` which btw. defaults to `Info.`